packages/apollo-datasource-rest/src/__tests__/RESTDataSource.test.ts

```diff
@@ -349,8 +349,8 @@ describe('RESTDataSource', () => {
 
       expect(fetch.mock.calls.length).toEqual(1);
       expect(fetch.mock.calls[0][0].url).toEqual('https://api.example.com/foo');
-      expect(fetch.mock.calls[0][0].body).not.toEqual('{}');
-      expect(fetch.mock.calls[0][0].headers.get('Content-Type')).not.toEqual(
+      expect(fetch.mock.calls[0][0].body).toEqual('{}');
+      expect(fetch.mock.calls[0][0].headers.get('Content-Type')).toEqual(
```

Comment:
Why are you reversing what these two expectations are testing for?

Reply:
@abernix see my comment below.